src/lib/mongoBackend/MongoGlobal.h

```diff
@@ -300,16 +300,12 @@ extern EntityIdVector subToEntityIdVector(const orion::BSONObj& sub);
 *
 * subToNotifyList -
 */
-void subToNotifyList
+void  subToNotifyList
 (
-  const std::vector<std::string>&  modifiedAttrs,
-  const std::vector<std::string>&  conditionVector,
   const std::vector<std::string>&  notificationVector,
   const std::vector<std::string>&  entityAttrsVector,
   StringList&                      attrL,
-  const bool&                      blacklist,
-  bool&                            op
-);
+  const bool&                      blacklist);
```

Comment:
Lint

```suggestion
  const bool&                      blacklist
);
```

Reply:
Fixed in [3ba1ead](https://github.com/telefonicaid/fiware-orion/pull/4310/commits/3ba1ead3128937e80932ee1f3210bacc23f3d475)